src/profile/main/tag/Google AdWords Remarketing - Prod_QA-UID26.js

```diff
@@ -66,11 +67,25 @@ try {
                             u.data.params[prefix] = u.data.params[prefix] || {};
                             u.data.params[prefix][e[f].substr(prefix.length + 1)] = b[d];
                         } else if (e[f].indexOf("custom.") == 0) {
+                            if (e[f].substr(7) === "isFlightConfirmation") {
```

Comment:
Maybe best to declare 'custom' as a constant PREFIX and use length instead of 7. Or even compare including the prefix. The 7 looks confusing...